src/views/components/checkbox/index.njk

```diff
@@ -0,0 +1,29 @@
+---
+title: Checkbox
+description: Checkboxes let the user select one or more options from a list.
+layout: default
+eleventyNavigation:
+  key: Checkbox
+  section: components
+---
+{% from "example.njk" import example %}
+
+<section>
+    <p>Checkboxes let the user select one or more options from a list.</p>
+    {{ example({group: "components", item: "checkbox", example: "default", size: "m", html: true, open: false }) }}
+</section>
+
+<section class="my-8">
+    <h2 class="mb-4">When to use this component</h2>
+    <p>Use the input component when you need to let users enter alphanumeric information that is no longer than one line, such as their name, telephone number or email address.</p>
+</section>
+
+<section class="mb-8">
+    <h2 class="mb-4">When not to use this component</h2>
+    <p>Do not use the checkboxes component if users can only choose one option from a list. In this case, you should use the <a href="/components/radios">radios</a> component.</p>
```

Comment:
Radios don't yet exist.

Reply:
(at least not in this pattern lib version)

Reply:
As the design system isn't currently in use then I'm inclined to leave this copy as is for now, because Nick is currently working on adding radios to the system and they should be incorporated imminently. Let me know your thoughts